Assets/_Functionalities/ObjectInformation/Scripts/BagInspector.cs

```diff
@@ -284,13 +284,10 @@ private void RenderThumbnailForFeature(FeatureMapping mapping)
             {
                 GeoJSONPolygonLayer polygonLayer = mapping.VisualisationLayer as GeoJSONPolygonLayer;
                 List<Mesh> meshes = mapping.FeatureMeshes;
-                for (int j = 0; j < meshes.Count; j++)
-                {
-                    PolygonVisualisation pv = polygonLayer.GetPolygonVisualisationByMesh(meshes);
-                    Bounds currentObjectBounds = new Bounds(pv.transform.position, meshes[j].bounds.size);
-                    featureThumbnail.RenderThumbnail(currentObjectBounds);
-                    break;
-                }
+                
+                PolygonVisualisation pv = polygonLayer.GetPolygonVisualisationByMesh(meshes);
+                Bounds currentObjectBounds = new Bounds(pv.transform.position, meshes[0].bounds.size);
+                featureThumbnail.RenderThumbnail(currentObjectBounds);
```

Comment:
hier gebeurt nu wat anders?

Reply:
Helaas niet - dat is precies de warning. Na de renderThumbnail in de vorige versie zat een `break` statement. Wat ervoor zorgt dat je direct de loop uitgedonderd wordt. Dus in de praktijk kwam het neer op deze statement. De for-loop was onnodig